content/en/docs/quick-start/all-in-one-on-linux.md

```diff
@@ -186,3 +186,6 @@ After logging in the console, you can check the status of different components i
 ## Enable Pluggable Components (Optional)
 
 The guide above is used only for minimal installation by default. To enable other components in KubeSphere, see [Enable Pluggable Components](../../pluggable-components/) for more details.
+
+## Demo
+<script src="https://asciinema.org/a/362291.js" id="asciicast-362291" async></script>
```

Comment:
@liuboaibc  The Demo looks not work, see the preview page: https://deploy-preview-344--kubesphere-v3.netlify.app/docs/quick-start/all-in-one-on-linux/

![image](https://user-images.githubusercontent.com/40452856/95574635-e4b26700-0a5f-11eb-992d-cbebef05d41d.png)

We hope the video and its player can be integrated into the website, example: https://asciinema.org/

Reply:
The code of this pr is not the latest code.